### ISSUE TITLE
Examples still show C++/CX code after update to C++/WinRT

### ISSUE THREAD

**Issue by pjmlp** [MicrosoftDocs/windows-uwp#2563]
I just noticed that the example was finally updated to C++/WinRT, thanks for the effort.

However the code snippet on this page still makes use of `ref new` from C++/CX.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 57ba16b2-6122-0499-b61d-ba93bf2781cf
* Version Independent ID: 9fb03c83-3f5c-dc72-09d6-0b571d2cf86d
* Content: [Extend the sample game - UWP applications](https://docs.microsoft.com/en-us/windows/uwp/gaming/tutorial-resources)
* Content Source: [windows-apps-src/gaming/tutorial-resources.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/gaming/tutorial-resources.md)
* Product: **uwp**
* Technology: **games**
* GitHub Login: @stevewhims
* Microsoft Alias: **stwhi**

**Comment by stevewhims** [comment]
Hi, @pjmlp, and thanks for filing this issue. For the workstream that I completed recently, I updated any topic that shows source code from the **Simple3DGameDX** sample game. This particular topic (the "extend" topic) doesn't show source code from that sample, but from another, which hasn't been ported yet. So that'll be a new workstream for me (plus, there are a few other topics that show C++/CX source code, not directly related to the **Simple3DGameDX** sample game). If it's ok, I'll handle this like I did the previous workstream, and track the work via my backlog. Hence, I'll close this issue so that I don't track it twice.

Thanks again!
Steve
